common/dbus/platform/asio_libdbus/dbus.cpp

```diff
@@ -429,7 +436,7 @@ ExpectedStringPair ExtractValueFromDBusMessage(DBusMessage *message) {
 		dbus_error_free(&dbus_error);
 		return expected::unexpected(err);
 	}
-	return std::pair<string, string> {string(value1), string(value1)};
+	return std::pair<string, string> {string(value1), string(value2)};
```

Comment:
This commit makes it sound like a new feature, but is this really a fix which adds the pieces that were missing?

Reply:
I'm splitting it into two commits -- a `fix` one and a `feat` one.